### PR TITLE
feat: Replace old gallery cards on profile

### LINKED ISSUE #5016: Replace old cards on profile
image
- old cards 

image
- new ones

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [x] Refactoring

## Context

- [x] Closes #5016
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

/bsx/u/bXjrHpyJzSczJuuqmtN6nqCnd5b7KFa5LK1rBtF1tUHDYzp8N

image

image

### CODE REVIEW

File: components/rmrk/Gallery/PaginatedCardList.vue [hunk 1]
```diff
@@ -44,7 +39,8 @@ import { Debounce } from 'vue-debounce-decorator'
 import shouldUpdate from '~/utils/shouldUpdate'
 
 const components = {
-  GalleryCardList: () => import('./GalleryCardList.vue'),
+  GalleryNftCardList: () =>
```
Comment (vikiival) [inline components/rmrk/Gallery/PaginatedCardList.vue:42]: GalleryItemCardList

Comment (Jarsen136) [reply components/rmrk/Gallery/PaginatedCardList.vue]: done

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 38.49 USD/KSM ~ 1.299 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xade12dde56fb24e77ff9fa75a22b91e36ee94de096e1a2992127186f26b209ad](https://kusama.subscan.io/extrinsic/0xade12dde56fb24e77ff9fa75a22b91e36ee94de096e1a2992127186f26b209ad)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**